forcedphot.rst

```diff
@@ -0,0 +1,181 @@
+
+The forced source tables exist on a per-patch basis. While each tract has 49 patches of data, loading even the bare minimum of columns into memory (via pandas for analsis in a jupyter notebook, for instance) is prohibitive for more than a few patches at a time.
+As a result, the most data we aggregated at one time is from 5 patches for one of the shallower tracts.
+Distributions of the forced PSF flux for all sources in patches 0--4 of tract 3898 are shown in :numref:`fig-flux-hist-3828`.
+We examined a single patch's forced difference flux for all sources as well, shown in :numref:`fig-diff-flux-hist-3828-47`.
+Nothing unusual is apparent in these plots, but they also do not provide the kind of granular object-level light curved vetting desired.
+
+.. figure:: _static/flux-hist-3828.png
+    :name: fig-flux-hist-3828
+    :target: ../_images/flux-hist-3828.png
+    :scale: 40 %
+    :alt: Six panels with flux histograms by band.
+
+    Forced flux distributions by band for all sources in tract 3828, patches 0--4.
+
+.. figure:: _static/diff-flux-hist-3828-47.png
+    :name: fig-diff-flux-hist-3828-47
+    :target: ../_images/diff-flux-hist-3828-47.png
+    :scale: 40 %
+    :alt: Six panels with difference flux histograms by band.
+
+    Forced flux distributions by band **on the difference images** for all sources in tract 3828, patch 47. Because most detected sources are not also difference imaging sources, most have a flux at or near zero.
+
+The memory problem is particularly pronounced when creating and examining light curves.
+For example, it is straightforward to write down metrics one may use to characterize light curve variability, or to search by sky coordinate for a known variable object of interest.
+It is much more challenging to meaningfully examine light curves for a representative sample of objects.
+
+.. _section-light-curve-sample:
+
+A (very small) sampling of light curves
+---------------------------------------
+
+We created a large pandas dataframe for a few patches of each tract by merging forced source table columns ``forcedSourceId`` (the index), ``band``, ``psfFlux``, ``psfFluxErr``, and ``objectId`` with CCD visit table column ``expMidpt`` by merging on the shared column ``ccdVisitId``.
+We then identified all unique objects and plotted light curves for a subset of them.
+Here we reproduce five light curves from each of three tracts. All y-axis values are force PSF fluxes in units of nJy.
+
+Tract 3828
+^^^^^^^^^^
+
+.. figure:: _static/lc1-3828.png
+    :name: fig-lc1-3828
+    :target: ../_images/lc1-3828.png
+
+.. figure:: _static/lc2-3828.png
+    :name: fig-lc2-3828
+    :target: ../_images/lc2-3828.png
+
+.. figure:: _static/lc3-3828.png
+    :name: fig-lc3-3828
+    :target: ../_images/lc3-3828.png
+
+.. figure:: _static/lc4-3828.png
+    :name: fig-lc4-3828
+    :target: ../_images/lc4-3828.png
+
+.. figure:: _static/lc5-3828.png
+    :name: fig-lc5-3828
+    :target: ../_images/lc5-3828.png
```

Comment:
lc5-3828 and lc3-3828 are rather exciting and seem to be varying a lot (I'm assuming the error bars are very small?). Since the goal here is to distinguish between "instrumental, bad" variation and "astrophysical, good" variation, it's worth giving some thought and words to how we might make that distinction for these light curves. (or to put it another way, what conclusion should we draw from seeing these?)

Reply:
Agreed. I added a few words about how actual flux variations and small error bars are the kinds of features we'll want to single out.